astroid/nodes/scoped_nodes/scoped_nodes.py

```diff
@@ -721,14 +719,13 @@ def __init__(
             parent=parent,
         )
 
-    def postinit(self, elt=None, generators=None):
+    def postinit(self, elt=None, generators: Optional["nodes.Comprehension"] = None):
```

Comment:
So it wasn't a List of Comprehension but a Comprehension ? (I.e. old value was wrong ?)

Reply:
No, this is a mistake.. 😅